receiver/sapnetweaverreceiver/metadata.yaml

```
@@ -30,102 +37,174 @@ metrics:
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.host.cpu_utilization:
-    description: CPU Utilization Percentage.
+    warnings:
+      if_enabled_not_set: This metric will be disabled by default soon.
+  sapnetweaver.host.cpu.utilization:
+    description: The CPU utilization percentage.
     unit: "%"
     gauge:
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.host.memory.virtual.overhead:
-    description: Virtualization System Memory Overhead.
-    unit: "bytes"
+  sapnetweaver.system.availability:
+    description: The system availability percentage.
+    unit: "%"
     gauge:
       value_type: int
+      input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.host.memory.virtual.swap:
-    description: Virtualization System Swap Memory.
-    unit: "bytes"
+  sapnetweaver.system.utilization:
+    description: The system utilization percentage.
+    unit: "%"
     gauge:
       value_type: int
+      input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.http.count:
-    description: The number of HTTP Sessions.
-    unit: "{sessions}"
+  sapnetweaver.memory.usage:
```

```suggestion
  sapnetweaver.memory.utilization:
```

This sounds like a utilization rather than a usage https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md

---

yeah, will update. I think maybe `sapnetweaver.memory.swap_space.utilization` may be better?